Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -154,14 +198,37 @@ const ContestsProvider = ({ children }: IContestsProviderProps) => {
         [ changePage, params, setParam, unsetParam ],
     );
 
+    const toggleSorting = useCallback(
```

Comment:
Same here. Almost identical to `toggleFilter`. Make a third function to do this stuff, i.e. `toggleParam` or something else, and `toggleFilter` & `toggleSorting` should call `toggleParam`

Reply:
I made a single function toggleParam and removed the other two.